### DISCUSSION TITLE
command not working

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1968] [General]
foundryup-zksync  command not working in my machine I'm
![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/85685981/c2aa3786-0b8b-426a-a68a-9f41c7cbdcc6)
 in wsl and have been using foundry since long time

**Comment by EngrPips** [comment]
Hello @panditdhamdhere, Please follow the `Quick Install` section of this `https://github.com/matter-labs/foundry-zksync` repo to install `foundry-zkSync` after you successfully install it run `foundryup-zksync` and it should work.

**Reply by panditdhamdhere** [reply]
> Hello @panditdhamdhere, Please follow the `Quick Install` section of this `https://github.com/matter-labs/foundry-zksync` repo to install `foundry-zkSync` after you successfully install it run `foundryup-zksync` and it should work.

It's really helpful ✅
